docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -77,70 +85,107 @@ feedbackIntegration({
+| `buttonBackgroundHover`   | `--button-background-hover`    | `#f6f6f7`                                 | `#352f3b`                                 | Background color when hovering over the cancel button                                                 |
+| `buttonBorder`            | `--button-border`              | `var(--border)`                           | `var(--border)`                           | Border style for the cancel button                                                                    |
+| `buttonBorderRadius`      | `--button-border-radius`       | `var(--input-border-radius)`              | `var(--input-border-radius)`              | Border radius style for form buttons                                                                  |
+| `buttonOutlineFocus`      | `--button-outline-focus`       | `var(--input-outline-focus)`              | `var(--input-outline-focus)`              | Outline color for the cancel button, in the focused state                                             |
+| `buttonForeground`        | `--button-foreground`          | `inherit`                                 | `inherit`                                 | Foreground color for the cancel button                                                                |
+| `buttonForegroundHover`   | `--button-foreground-hover`    | `inherit`                                 | `inherit`                                 | Foreground color for the cancel button when hovering                                                  |
+
 | `submitBackground`        | `--submit-background`          | `rgba(88, 74, 192, 1)`                    | `rgba(88, 74, 192, 1)`                    | Background color for the submit button                                                                |
 | `submitBackgroundHover`   | `--submit-background-hover`    | `rgba(108, 95, 199, 1)`                   | `rgba(108, 95, 199, 1)`                   | Background color when hovering over the submit button                                                 |
 | `submitBorder`            | `--submit-border`              | `rgba(108, 95, 199, 1)`                   | `rgba(108, 95, 199, 1)`                   | Border style for the submit button                                                                    |
-| `submitOutlineFocus`      | `--submit-outline-focus`       | `rgba(108, 95, 199, 1)`                   | `rgba(108, 95, 199, 1)`                   | Outline color for the submit button, in the focused state                                             |
+| `submitBorderRadius`      | `--submit-border-radius`       | `var(--button-border-radius)`             | `var(--button-border-radius)`             | Border radius style for submit buttons                                                                |
```

Comment:
I don't think we should allow configuration of the submit border radius specifically. I think this should only be configured with the other button border radiuses

Reply:
that's fair

Reply:
it defaults to `var(--button-border-radius)` right now, we'll get it fixed up though.

Reply:
removed references in the docs to all the border-radius overrides. 👍